panther_docking/test/unit/test_panther_charging_dock.cpp

```
@@ -0,0 +1,209 @@
+    const std::string & frame_id, const std::string & child_frame_id,
+    const builtin_interfaces::msg::Time & stamp, const geometry_msgs::msg::Transform & transform);
+
+  rclcpp_lifecycle::LifecycleNode::SharedPtr node_;
+  std::shared_ptr<PantherChargingDockWrapper> dock_;
+  rclcpp::Publisher<geometry_msgs::msg::PoseStamped>::SharedPtr dock_pose_pub;
+  tf2_ros::Buffer::SharedPtr tf_buffer_;
+};
+
+TestPantherChargingDock::TestPantherChargingDock()
+{
+  node_ = std::make_shared<rclcpp_lifecycle::LifecycleNode>("test_node");
```

cppcheck

[performance] Variable 'node_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)